src/contracts/pods/EigenPod.sol

```diff
@@ -32,7 +33,8 @@ contract EigenPod is
     Initializable, 
     ReentrancyGuardUpgradeable, 
     EigenPodPausingConstants, 
-    EigenPodStorage 
+    EigenPodStorage,
+    Test
```

Comment:
Oh, we should definitely remove Test artifacts!

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/591/commits/20bf8100b15a60ae01f3319eac1a97ebea1a13b7